README.md

```diff
@@ -4,17 +4,17 @@ A reference library for building Javascript applications on top of the Enceeper
 
 ## Introduction
 
-The [Enceeper app](https://github.com/enceeper/enceeper) and the [Enceeper service](https://www.enceeper.com/) can be used to securely store and retrieve credentials (usernames, passwords, API keys etc). The Enceeper app can be divided into two parts:
+The [Enceeper app](https://github.com/enceeper/enceeper) and the [Enceeper service](https://www.enceeper.com/) are used to securely store and retrieve credentials (usernames, passwords, API keys etc). The Enceeper app is divided into two parts:
 - The User Interface (UI) that handles all user input
 - The core functionality and the utilization of the Enceeper service
 
-For the later we have created this repository, to make it a seperate project for easier integration (via npm) to other solutions.
+For the latter, use this repository which serves as a basis project for easier integration (via npm) to other solutions.
 
-The purpose of this library is:
+Purpose of this library is:
 * To execute the SRP6a protocol both for user registration and authentication (https://github.com/alax/jsrp)
-* Implement all cryptographic functionality by utilizing [SJCL](https://github.com/bitwiseshiftleft/sjcl) and [TweetNaCl](https://github.com/dchest/tweetnacl-js)
+* Implement full cryptographic functionality, utilizing [SJCL](https://github.com/bitwiseshiftleft/sjcl) and [TweetNaCl](https://github.com/dchest/tweetnacl-js)
```

Comment:
"all" is better here

Reply:
fine by me..still removing "by" i think is valid.